config.example.yml

```diff
@@ -79,6 +79,15 @@ retention:
   # Number of days to retain historical data for SLA reporting. By default, it is retained forever.
 #  sla-days:
 
+  # Interval for periodically cleaning up the historical data, defined as a duration string.
+  # A duration string is a sequence of decimal numbers and a unit suffix, such as "20s".
+  # Valid units are "ms", "s", "m", "h". Defaults to "1h".
```

Comment:
Totally pedantic, but could you please add a newline between the list of valid units and the default to make the default value stand out more.